frontends/election-manager/src/components/import_cvrfiles_modal.tsx

```diff
@@ -143,7 +143,14 @@ export function ImportCvrFilesModal({ onClose }: Props): JSX.Element {
   async function importSelectedFile(
     fileData: CastVoteRecordFilePreprocessedData
   ) {
-    await importCvrFile(new File([fileData.fileContent], fileData.name));
+    assert(window.kiosk);
+    // TODO: To avoid unnecessarily loading potentially large CVR files into
+    // memory, we might want to add a file streaming API to kiosk-browser, or
+    // update the backend API to support importing files with just a file path
+    // instead of file attachments.
```

Comment:
Agreed, I think it makes sense for the frontend to only know about file paths - it doesn't need to load file contents

Reply:
In fact, I'd go so far as to say the backend should be the one that reads the USB drive and gives the frontend just enough info to render the table and communicate the selections back to the server. Maybe that's file paths, but maybe not even that!